pytab/core.py

```diff
@@ -361,6 +362,13 @@ def cli(
     click.echo("| Obtaining System Information...", nl=False)
     system_info = hwi.get_system_info()
     click.echo(" success!")
+    click.echo("| Detected System Config:")
+    click.echo(yaml.dump(system_info, default_flow_style=False))
+    sysinfo_input: str = click.prompt("Is this correct? [Y/n]", type=str, default='y', show_default=False, show_choices=False)
+    if sysinfo_input is not None and sysinfo_input.lower() == 'n':
+        click.echo("ERROR: Incorrect system info, aborted by user.")
```

Comment:
Why abort, if hwinfo is wrong?
Only reason to display the info is so that the user can correct it.

If something is detected wrong, the main script will not run any tests with it anyways.
Hence pointless abborting the script manualy.